src/main/java/Duke.java

```diff
@@ -1,10 +1,193 @@
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import Command.*;
+import Entities.*;
+import Exceptions.*;
+import FileUtils.*;
```

Comment:
One suggestion is to avoid .* imports.

Reply:
Thank you! Will change it away from wildcard import!